dbux-code/src/globalAnalysisView/nodes/GlobalErrorsNode.js

```diff
@@ -33,6 +36,12 @@ export default class GlobalErrorsNode extends BaseTreeViewNode {
         return child;
       }
     }
+    for (const child of this.children) {
```

Comment:
can use `this.children.find` instead

Reply:
Cannot simply use `find` since it is a `flatFind`(or `findMap`) operation.